docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  toxiproxy:
+    image: ghcr.io/shopify/toxiproxy
+    ports:
+      - "8474:8474"
+
+  ci-tests:
+    build:
+      context: .
+      target: dev
+    image: toxiproxy-php-client
+    command: ["composer", "test"]
+    environment:
+      TOXIPROXY_HOST: toxiproxy
+    depends_on:
+      toxiproxy:
+        condition: service_started
```

Comment:
Jak to vyvyjist lokalne? Je to libka, tak tohle stalo za to uz uplne otocit, kdyz se to tu dela na cerstvo (jak to delame v platform libs):
* zrusit `ci-tests`
* zavest `dev` s mountovanymi zdrojaky
* `Dockerfile` bez zdrojaku/`composer.*`
* v CI poustest `docker comose run --rm dev bash -c "composer install && composer test"